### PR TITLE
Client email changes

### PR DESCRIPTION
**Pull request by deepti-kakade**
**Notion:**

https://www.notion.so/saeloun/Backend-Add-new-client-form-should-support-multiple-emails-d30681a5f0e94742a249de1950ada1f8

**Description:**

- Backend is allowing multiple emails for a client. 
- Saving emails as an array in the database.
- Replaced Ransack with Searchkick for the Client.
- Added Searchkick in the client model.
- Updated test cases.

**Screenshots:**

**Note:** Frontend changes are required. In this PR we handled backend changes.

### CODE REVIEW

File: app/services/clients/index_service.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        client_details:,
+        total_minutes:,
+        overdue_outstanding_amount:
+      }
+    end
+
+    private
+
+      def clients
+        if params[:q].present?
```
Comment (prasanthchaduvula) [inline app/services/clients/index_service.rb:25]: Please make it into a query for more readability `params[:query]`

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

**Review by prasanthchaduvula — COMMENTED** [review]
Backend changes LGTM

**Comment by supriya3105** [comment]
@mayankagnihotri7 Copy the file changes in another PR and raise the PR assign to Shruti so that she can work on the front end changes.

**Comment by mayankagnihotri7** [comment]
Closing this PR. Raised a new one which includes the latest changes from the develop branch.

**Pull request closed without merging**